bpf/lib/proxy.h

```diff
@@ -56,6 +56,23 @@ ctx_redirect_to_proxy(struct __ctx_buff *ctx, __be16 proxy_port)
 #endif
 }
 
+/**
+ * ctx_redirect_to_proxy_first() applies changes to the context to forward
+ * the packet towards the proxy. It is designed to run as the first function
+ * that accesses the context from the current BPF program.
+ */
+static __always_inline int
+ctx_redirect_to_proxy_first(struct __ctx_buff *ctx, __be16 proxy_port)
+{
+	int ret = CTX_ACT_OK;
+
+	cilium_dbg(ctx, DBG_CAPTURE_PROXY_POST, proxy_port, 0);
+	ctx->mark = MARK_MAGIC_TO_PROXY | (proxy_port << 16);
+	ctx_change_type(ctx, PACKET_HOST);
+
+	return ret;
```

Comment:
```suggestion
	return CTX_ACT_OK;
```
And let's get rid of `ret`? Or why not even returning `void`?

Reply:
These changes are in context of other changes I'll make. I can always apply this feedback now but I will change it again in the eBPF tproxy PR. That's probably fair though anyway. Will address.